app.js

```diff
@@ -5,11 +5,14 @@ var app = express();
 var Server;
 var http = require('http');
 var https = require('https');
-var storageModule = require('storage');
+var storageModule = require('./storage');
+var fs = require('fs');
 
 var PORT = process.env.PORT || 8080;
 
-storageModule.init(config);
+app.use(express.static('views'));
+
+//storageModule.init(config);
```

Comment:
Why Storage Connection Was not instantiated. Please uncomment this line and make use of the promise returned by this function to start Listening From the Server only when Instatiation is successfull and it goes to otherwise(err) then Exit with proper error message. Include log.js to write these logs.